jobs/build-arch.Jenkinsfile

```diff
@@ -539,7 +510,7 @@ EOF
             }
 
             echo message
-            if (notify_slack == "yes") {
+            if (official) {
```

Comment:
Meh FWIW, in the spirit of keeping the minimum amount of things gated on `official`, I'd just leave the `notify_slack` knob. It's also not entirely unreasonable to have someone testing changes to Slack messages and targeting their own private channel.

But also OK with this. In that case though, we should also delete the `notify_slack` variable.

Reply:
yeah I went and cleaned up the extraneous vars. Thanks for catching them.